### ISSUE TITLE
When more than 9 script are imported to minimize, cache doesn´t work. beta 1.2.4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#206]
```
Minify version:
PHP version: PHP Version 5.2.8

What steps will reproduce the problem?
1. <script type="text/javascript" 
src="/min/b=Aedit/js&amp;f=jquery-1.4.3.js,jquery.metadata.js,jquery.selectboxes
.min.js,jquery.validate.js,localization/messages_de.js,jquery.effects.highlight.
js,jquery.easydrag.js,Aedit.js,print_r-min.js,Aedit.Ajax.js"></script>
10 js files get imported, files get not cached. Load time about 4s

2. <script type="text/javascript" 
src="/min/b=Aedit/js&amp;f=jquery-1.4.3.js,jquery.metadata.js,jquery.selectboxes
.min.js,jquery.validate.js,localization/messages_de.js,jquery.effects.highlight.
js,jquery.easydrag.js,Aedit.js,print_r-min.js"></script>
9 js files are imported. load time 16 ms

I tested this with many different files (and filesizes).

```

---

Original issue reported on code.google.com by `Smike...@gmail.com` on 14 Nov 2010 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
Good catch, I'll look into this soon.
```

---

Original comment by `mrclay....@gmail.com` on 14 Nov 2010 at 7:11
- Changed state: **Accepted**
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Is there any possibility you have your debug cookie set to match Aedit.Ajax.js 
? When you call the URL directly do you get minified output?
```

---

Original comment by `mrclay....@gmail.com` on 14 Nov 2010 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
debug cookie? dont have set such thing.
i get minified output, when entering following url:
http://127.0.0.1/min/b=Aedit/js&f=jquery-1.4.3.js,jquery.history.js,jquery.form.
js,jquery.selectboxes.min.js,jquery.validate.js,jquery.metadata.js,localization/
messages_de.js,jquery.purr.js,jquery.effects.core.js,jquery.effects.highlight.js
it just loads ~ 4s.

http://127.0.0.1/min/b=Aedit/js&f=jquery-1.4.3.js,jquery.history.js,jquery.form.
js,jquery.selectboxes.min.js,jquery.validate.js,jquery.metadata.js,localization/
messages_de.js,jquery.purr.js,jquery.effects.core.js
Shows instantly.
Does this bug appear only on my side?
```

---

Original comment by `Smike...@gmail.com` on 14 Nov 2010 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
Can you e-mail a zip of those files to steve@mrclay.org? I haven't tested, but 
I have a feeling something is causing the cacheId to change every request.
```

---

Original comment by `mrclay....@gmail.com` on 15 Nov 2010 at 1:22

**Comment by GoogleCodeExporter** [comment]
```
Just tested and can't reproduce this bug on Win XP with latest XAMPP (PHP 
5.3.1/Apache 2.2.14). I think you're going to need to walk through a request 
using remote debugging (xdebug) to figure out what's going on. I'm happy to 
help if you're willing to put the effort into it. At the very least, watch your 
cache directory and see what happens as you make repeated requests.
```

---

Original comment by `mrclay....@gmail.com` on 16 Nov 2010 at 5:17
- Changed state: **NeedInfo**
- Added labels: **Priority-Medium**
- Removed labels: **Priority-Critical**

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `mrclay....@gmail.com` on 3 Sep 2011 at 11:45
- Changed state: **Invalid**
